mne/viz/topomap.py

```diff
@@ -709,8 +710,10 @@ def _plot_topomap(data, pos, vmin=None, vmax=None, cmap=None, sensors=True,
     if (Zi == Zi[0, 0]).all():
         cont = None  # can't make contours for constant-valued functions
     else:
-        cont = ax.contour(Xi, Yi, Zi, contours, colors='k',
-                          linewidths=linewidth / 2.)
+        with warnings.catch_warnings(record=True):
```

Comment:
just to start a conversation. Whats your opinion on using pytest and asserts in the codebase outside of testing code? In other words, how you argue to not use pytest if by the end of the day is nicer to use. 

And also why not using something like this
```diff
def xx(a=0, b=10):
-     if a > b:
-          raise(ValueError, f"a={a} should be smaller than b={b}")
+     assert a < b, f"a={a} should be smaller than b={b}"
```